source/agora/node/Ledger.d

```diff
@@ -112,7 +112,7 @@ public class Ledger
         this.enroll_man = enroll_man;
         this.pool = pool;
         this.onValidatorsChanged = onValidatorsChanged;
-        if (!this.storage.load())
+        if (!this.storage.load(GenesisBlock))
```

Comment:
Shouldn't the Node pass the configured genesis block to the ledger, so it can be used here?

Reply:
In other words, shouldn't it be a ctor parameter?

Reply:
I thought about it, but it meant having a useless field for the lifetime of the instance, since it's only ever used for `load`.
It felt like a bad design, but if you prefer I can amend the code.

Reply:
Ah right this is not a "constant" here, it's changed in the call to setGenesisBlock.